Bottom: c409939610256c83cdcf4b4ccec1db8ec5a38f1f
Top:    7733f5da8e20ed38265871f63f2836df06985a1d
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-04-25 15:50:19 +0100

draw.c: Remove API ClearAndRedrawOutput(), in favour of Redraw().

These do the same thing anyway.


---

diff --git a/src/draw.h b/src/draw.h
index d62ed7d..adfee36 100644
--- a/src/draw.h
+++ b/src/draw.h
@@ -37,7 +37,6 @@
 
 void Draw (void);
 void Redraw (void);
-void ClearAndRedrawOutput (void);
 void DrawVia (PinTypePtr);
 void DrawRat (RatTypePtr);
 void DrawViaName (PinTypePtr);
